docs/user/getting-started.md

```diff
@@ -102,8 +102,7 @@ flightctlUi:
   image: quay.io/flightctl/flightctl-ui:latest
   flightctlServer: https://flightctl-api:3443
   flightctlMetricsServer: https://flightctl-api:9090
-  bootcImgUrl: quay.io/example/example-agent-centos:bootstrap
-  qcow2ImgUrl: https://example.com/disk.qcow2
```

Comment:
I guess these are deprecated?

Reply:
These are not mandatory anymore. User could still provide them, but I dont want to complicate things in this getting started doc.